src/Microsoft.TestPlatform.CommunicationUtilities/LengthPrefixCommunicationChannel.cs

```diff
@@ -85,14 +85,10 @@ public Task NotifyDataAvailable()
             // Try read data even if no one is listening to the data stream. Some server
             // implementations (like Sockets) depend on the read operation to determine if a
             // connection is closed.
```

Comment:
```suggestion
            // Try read data even if no one is listening to the data stream. Some server
            // implementations (like Sockets) depend on the read operation to determine if a
            // connection is closed.
```

This comment is not accurate, we did not read it even before, and it would cause us to lose messages, or have to cache them until a subscriber appears. Instead of that we now have a backpressure